run_dir/static/js/pricing_quote.js

```diff
@@ -604,6 +604,20 @@ app.component('v-pricing-quote', {
                   <button type="submit" class="btn btn-primary" id="generate_quote_btn" v-on:click="generate_quote('save')" :disabled="this.invoice_downloaded"> Save Agreement</button>
                 </div>
               </div>
+              <div v-if="origin === 'Agreement'" class="ml-n1 mt-5">
+                <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
Panels are from the old version of bootstrap. The new one uses cards.

Reply:
Aha, I took it from here: https://github.com/SciLifeLab/genomics-status/blob/406ba246f8e89f003e15479adaad457942f84451/run_dir/design/pricing_quote.html#L41

Reply:
OK, how's this?